src/plugins/ResizeReorder/ResizerLine.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
Dummy Component to render a big blue line based on cursor position. It is rendered in ResizerKnob componenDummy Component to render a big blue line based on cursor position. It is rendered in ResizerKnob componen